### DISCUSSION TITLE
x-spread in Alpine 3

### DISCUSSION THREAD

**Discussion by nektobit** [alpinejs/alpine#1673] [1. Help]
I noticed that the x-spread directive was removed in alpine v3. Please, if anyone knows why it was removed (maybe there are links to the discussion), let me know about it, I am writing a short article about alpine.js where I compare the capabilities of versions 2 and 3.

**Comment by nektobit** [comment]
The magic property $event is also gone. If anyone has any comments on this, please post here.

**Reply by ryangjchandler** [reply]
Can you share an example of where this is no longer working please.

**Reply by nektobit** [reply]
@ryangjchandler most likely it continues to work, just not listed as a magic property in this documentation: https://alpinejs.dev/alpine-101

In documentation v2 this is explicitly indicated https://github.com/alpinejs/alpine/tree/v2.8.2#event

**Reply by SimoTod** [reply]
It's documented in the "events" section: https://alpinejs.dev/essentials/events#accessing-the-event-object

**Comment by ryangjchandler** [comment]
@nektobit `x-spread` has been replaced with `x-bind`, here's how to use it:

```
<div x-bind="yourOldSpreadObjectHere">
</div>
```

**Reply by nektobit** [reply]
Thanks a lot for the clarification!
